src/lib/roles.ts

```diff
@@ -196,7 +237,8 @@ export const roles_definition = validateRoles({
 			"0":"CLOSED",
 			"1":"TILTED",
 			"2":"OPEN"
-		}
+		},
+		"unit": "null"
```

Comment:
Shouldn't this be `null`, not `"null"`?

Reply:
Null is now allowed due to Typescript checking :-)

"Type 'null' is not assignable to type 'string | string[] | undefined'.

Anyway, we need a value of unit against which we can check if it's not allowed to set a unit.

We can choose a more descriptive name, e.g. "unallowed"

Reply:
> "Type 'null' is not assignable to type 'string | string[] | undefined'.

Then let's update our definition for the interface. I'd be fine with some more descriptive name instead of `null`.

Reply:
When we choose a more descriptive string instead of null we don't need to update the definition of the interface.

Should we choose "unallowed" ?

Reply:
`"disallowed"` or `"forbidden"` sound better IMO

Reply:
"forbidden" is perfect.
Gona change and create a new commit.